File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int Height;
    do
    {
        Height =  get_int("Height: ");
    }
    while (Height > 8 || Height < 1);
    for (int i = 1; i <= Height; i++)
    {
        //This loop will print spaces
        for (int k = 0; k < Height - i; k++)
        {
            printf(" ");    
        }
        //This loop will print #s
        for (int j = 1; j <= i; j++)
        {
            printf("#");
           
        }
        //prints the aisle between the two pyramids
        printf("  ");
        //prints the seconed pyramid
        for (int j = 1; j <= i; j++)
        {
            printf("#");
           
        }
      
        printf("\n");
    }
    
}

File: mario.py
# importing get_int function from cs50 library
from cs50 import get_int

# prompt user for input until he type it correctly (1-8)
while True:
    n = get_int("Height: ")
    if n < 9 and n > 0:
        break
# main for loop
for i in range(n):
    # this loop prints spaces
    for k in range(n-i-1):
        print(" ", end="")
    # this loop print #
    for j in range(i+1):
        print("#", end="")
    # this to print a newline
    print("")